src/main/scala/github/gphat/datadog/package.scala

```diff
@@ -7,5 +7,5 @@ package object datadog {
     tags: Option[Seq[String]] = None, host: Option[String]
   )
 
-  case class Response(statusCode: Int, body: String)
+  case class Response(statusCode: Int, body: String, headers: String)
```

Comment:
I think a `Map<String, String>` would be better for general purpose, but I guess `String` is OK for now if we just want it for debugging only.